docs/data/introduction/support/support.md

```diff
@@ -21,6 +21,25 @@ If you think you've found a bug, or you have a new feature idea:
 - Please don't group multiple topics in one issue.
 - Please don't comment "+1" on an issue. It spams the maintainers and doesn't help move the issue forward. Use GitHub reactions instead (👍).
 
+### Bug reproductions
+
+We require bug reports to be accompanied by a minimal reproduction.
```

Comment:
```suggestion
We require bug reports to be accompanied by at least a minimal reproduction.
```

Reply:
Disagree, doesn't add any useful info to the sentence, and we actually want the reproduction case to be as minimal as possible (as opposed to getting a full copy-paste of their entire project).

Reply:
Yes, right! Haven't thought about that! 🤔